index.ts

```diff
@@ -84,6 +102,10 @@ export function parse(input: string): IParseReturn {
   return ret;
 }
 
+/**
+ * Produces a URL query string from a given obj
+ * @param obj the object to be stringified
+ */
```

Comment:
seems the 2nd argument, `options` should be listed

could its definition be read when hovering on the function name in VSC?

Reply:
> could its definition be read when hovering on the function name in VSC?

nop unless we provide `@type` jsdoc